ui/App.svelte

```diff
@@ -17,6 +17,7 @@
   import Project from "./Screen/Project.svelte";
   import Projects from "./Screen/Projects.svelte";
   import RegisterProject from "./Screen/RegisterProject.svelte";
+  import RegisterUser from "./Screen/RegisterUser.svelte";
```

Comment:
This naming convention might not be ideal. It sounds like an active action instead of a concept. Concretely one **registers** the **user** on the `UserRegistration` screen. So I'd advocate for `UserRegistration` wich also groups the prefixes more naturally.